stdlib/math.jou

```diff
@@ -1,40 +1,112 @@
+# Math lib
+#TODO: add test
+
+# Other functions
+# Returns the absolute value of x.
 declare abs(x: int) -> int
 declare llabs(x: long) -> long
 declare fabs(x: double) -> double
+# Returns x*y+z.
+declare fma(x: double, y: double, z: double) -> double
 
+# Rounding and remainder functions
+# Rounds x upward, returning the smallest integral value that is not less than x.
 declare ceil(x: double) -> double
+# Rounds x downward, returning the largest integral value that is not greater than x.
 declare floor(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded towards zero):
+# fmod = numer - tquot * denom
+declare fmod(numer: double, denom: double) -> double
+# Rounds x toward zero, returning the nearest integral value that is not larger in magnitude than x.
+declare trunc(x: double) -> double
+# Returns the integral value that is nearest to x, with halfway cases rounded away from zero.
 declare round(x: double) -> double
+declare lround(x: double) -> long
+# Rounds x to an integral value,
+declare rint(x: int) -> double
+declare lrint(x: long) -> double
+# Rounds x to an integral value
+declare nearbyint(x: double) -> double
+# Returns the floating-point remainder of numer/denom (rounded to nearest):
+# remainder = numer - rquot * denom
+declare remainder(numer: double, denom: double) -> double
 
-declare acos(x: double) -> double
-declare asin(x: double) -> double
-declare atan(x: double) -> double
-declare atan2(y: double, x: double) -> double
+# Trigonometric functions
+# Returns the cosine of an angle of x radians.
 declare cos(x: double) -> double
+# Returns the sine of an angle of x radians.
 declare sin(x: double) -> double
+# Returns the tangent of an angle of x radians.
 declare tan(x: double) -> double
+# Returns the principal value of the arc cosine of x, expressed in radians.
+declare acos(x: double) -> double
+# Returns the principal value of the arc sine of x, expressed in radians.
+declare asin(x: double) -> double
+# Returns the principal value of the arc tangent of, expressed in radians.
+declare atan(x: double) -> double
+# Returns the principal value of the arc tangent of, expressed in radians.
+declare atan2(x: double, y: double) -> double
 
-declare fmod(x: double, y: double) -> double
-declare sqrt(x: double) -> double
-declare cbrt(x: double) -> double
+# Floating-point manipulation functions
+# Returns a value with the magnitude of x and the sign of y
+declare copysign(x: double, y: double) -> double
```

Comment:
Is there a use case for this function? I don't think I have ever used it in Python or C.

Reply:
like 
```python
printf("%d", copysign(4, -1))
```
And it will output -4.0
I will add an example for it.